### PR TITLE
Fix binary `or`

### PR DESCRIPTION
**Pull request by t-bltg**
This is a fix for https://docs.juliaplots.org/v1.25/generated/unicodeplots/#unicodeplots-ref53, where obviously a binary `or` between markers and a character from the canvas lookup table is wrong.

Fix https://github.com/JuliaPlots/UnicodePlots.jl/issues/211.

### CODE REVIEW

File: src/canvas/braillecanvas.jl [hunk 1]
```diff
@@ -93,8 +93,9 @@ function pixel!(c::BrailleCanvas, pixel_x::Int, pixel_y::Int, color::UserColorTy
     0 <= pixel_x <= c.pixel_width || return c
     0 <= pixel_y <= c.pixel_height || return c
     char_x, char_y, char_x_off, char_y_off = pixel_to_char_point(c, pixel_x, pixel_y)
-    c.grid[char_x, char_y] =
-        Char(UInt64(c.grid[char_x, char_y]) | UInt64(braille_signs[char_x_off, char_y_off]))
+    if 0x2800 <= (val = UInt64(c.grid[char_x, char_y])) <= 0x28ff
+        c.grid[char_x, char_y] = Char(val | UInt64(braille_signs[char_x_off, char_y_off]))
+    end
```
Comment (johnnychen94) [inline src/canvas/braillecanvas.jl:98]: Both versions are encrypted to be... 😢

Comment (t-bltg) [reply src/canvas/braillecanvas.jl:98]: Sorry, I didn't get that, what versions ?

Comment (johnnychen94) [reply src/canvas/braillecanvas.jl:98]: The master version and the PR version... I'm afraid that I can't review this PR.

Comment (t-bltg) [reply src/canvas/braillecanvas.jl:98]: Oh you mean you do not understand the changes ? Let me explain: `0x2800` is [blank braille](https://unicode-table.com/en/2800/), `0x28ff` is [full braille](https://unicode-table.com/en/28FF/). Doing a `binary or` is the low level operation for 'filling' the braille pattern with dots. So I'm just checking here that the value is a braille unicode char, before applying the increment.

Comment (t-bltg) [reply src/canvas/braillecanvas.jl]: I've clarified this by using `const`s.

**Review by johnnychen94 — APPROVED** [review]
The result definitely looks good to me but to be honest I don't understand the details here.

**Pull request merged**